pages/linux/faillock.md

```diff
@@ -0,0 +1,16 @@
+# faillock
+
+> Display and modify authentication failure record files.
+> More information: <https://github.com/linux-pam/linux-pam>.
```

Comment:
```suggestion
> More information: <https://manned.org/faillock>.
```

The current link doesn't give details  about faillock. Can you find any better resources or we will have to default to a man page.